Error 43 with \setmainlanguage

I get the following error and I don't know what I'm doing wrong. I'm just trying to compile the default invoice 

```
Error producing PDF.
! Argument of \str_uppercase:n has an extra }.
<inserted text>
                \par
l.61 \setmainlanguage{}

make: *** [makefile:6: output.pdf] Error 43
```

I wonder a bit why it says line 61, because \setmainlanguage is on line 69 in template.tex

--- a/details.yml
+++ b/details.yml
@@ -39,7 +39,7 @@ closingnote: |
 # Invoice settings
 currency: EUR
 # commasep: true
-lang: en-GB
+lang: english
 
 # Typography and layout
 seriffont: Hoefler Text

--- a/template.tex
+++ b/template.tex
@@ -65,8 +65,7 @@ $endif$
 % LANGUAGE
 %--------------------------------
 $if(lang)$
-\usepackage{polyglossia}
-\setmainlanguage{$polyglossia-lang.name$}
+\usepackage[$lang$]{babel}
 $endif$
 
 % PDF SETUP

